docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_meda:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
+
+  db:
+    image: postgres:16.0-alpine3.17
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_db:$PGDATA
+
+volumes:
+  my_db:
+  my_meda:
```

Comment:
The volume name `my_meda` seems to be a typo. It should likely be `my_media` to correctly represent the media files directory. Correct this to ensure proper volume usage.